Jenkinsfile

```diff
@@ -146,17 +145,25 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
             sh 'make defconfig'
         }
         stage('Build') {
-            // Attempt to build quickly and reliably
-            try {
-                sh "make -j4 V=s ${params.ALL_PACKAGES ? 'IGNORE_ERRORS=m' : ''}"
-            } catch (hudson.AbortException err) {
-                // TODO BUG JENKINS-28822
-                if(err.getMessage().contains('script returned exit code 143')) {
-                    throw err
+            // Add opkg signing key
+            withCredentials([
+                [$class: 'FileBinding', credentialsId: 'opkg-build-private-key', variable: 'PRIVATE_KEY'],
+                [$class: 'FileBinding', credentialsId: 'opkg-build-public-key', variable: 'PUBLIC_KEY'],
+            ]){
+                sh "ln -s ${env.PRIVATE_KEY} ${WORKSPACE}/key-build"
```

Comment:
Symbolically linked keys are not installed on target filesystem so opkg update is failing.

root@OpenWrt:/etc/opkg/keys# ls -al
drwxrwxr-x    2 root     root            63 Nov 28 12:58 .
drwxr-xr-x    1 root     root           232 Dec  1 04:59 ..
-rw-rw-r--    1 root     root           107 Nov 28 12:58 af22f7a88858c8e9
lrwxrwxrwx    1 root     root            26 Nov 28 12:58 cc2a8b8aff1e5f1e -> creator-opkg-build-key.pub